man/tpm2_verifysignature.1.md

```diff
@@ -0,0 +1,103 @@
+tpm2_verifysignature 1 "SEPTEMBER 2017" tpm2-tools
+==================================================
+
+NAME
+----
+
+tpm2_verifysignature(1) - Validates a signature using the TPM.
+
+SYNOPSIS
+--------
+
+`tpm2_verifysignature` [OPTIONS]
+
+DESCRIPTION
+-----------
+
+tpm2_verifysignature(1) uses loaded keys to validate a signature on a message
+with the message digest passed to the TPM. If the signature check succeeds,
+then the TPM will produce a `TPMT_TK_VERIFIED`. Otherwise, the TPM shall return
+`TPM_RC_SIGNATURE`. If _KEY\_HANDLE_ references an asymmetric key, only the
+public portion of the key needs to be loaded. If _KEY\_HANDLE_ references a
+symmetric key, both the public and private portions need to be loaded.
+
+OPTIONS
+-------
+
+  * `-k`, `--keyHandle`=_KEY\_HANDLE_:
+
+    Handle of key that will used in the validation.
+
+  * `-c`, `--keyContext`=_KEY\_CONTEXT\_FILE_:
+
+    Filename of the key context used for the operation.
+
+  * `-P`, `--pwdk`=_KEY\_PASSWORD_:
```

Comment:
This is not a valid option for the tool.

Reply:
Done